animation.html

```diff
@@ -0,0 +1,227 @@
+        -webkit-animation-iteration-count: 1;
+      }
+
+      @keyframes bluecar {
+        100% {
+          transform: translatex(-1500px);
+        }
+      }
+
+      .red-car {
+        left: 50px;
+        position: absolute;
+        bottom: 190px;
+        z-index: 3;
+      }
+
+      .red-car img {
+        width: 100%;
+        height: 150px;
+        animation: car 5s linear infinite;
+        -webkit-animation-iteration-count: 1;
+      }
+
+      @keyframes car {
+        100% {
+          transform: translatex(1500px);
+        }
+      }
+
+      .wheel {
+        left: 50px;
+        position: absolute;
+        bottom: 200px;
+        z-index: 3;
+      }
+
+      .wheel img {
+        width: 40px;
+        height: 40px;
+        animation: wheel 5s linear infinite;
+        -webkit-animation-iteration-count: 1;
+      }
+
+      .back-wheel {
+        left: 40px;
+        bottom: 5px;
+        position: absolute;
+      }
+
+      .front-wheel {
+        left: 238px;
+        bottom: 5px;
+        position: absolute;
+      }
+
+      @keyframes wheel {
+        100% {
+          transform: translatex(1500px) rotate(360deg);
+        }
+      }
+
+      .right-car-wheel {
+        left: 75%;
+        position: absolute;
+        bottom: 200px;
+        z-index: 2;
+      }
+
+      .right-car-wheel img {
+        width: 40px;
+        height: 40px;
+        animation: bluewheel 5s linear infinite;
+        -webkit-animation-iteration-count: 1;
+      }
+
+      .back-wheel-blue {
+        left: 39px;
+        bottom: 20px;
+        position: absolute;
+      }
+
+      .front-wheel-blue {
+        left: 220px;
+        bottom: 20px;
+        position: absolute;
+      }
+
+      @keyframes bluewheel {
+        100% {
+          transform: translatex(-1500px) rotate(-360deg);
+        }
+      }
+
+      .middle-car {
+        left: 45%;
+        position: absolute;
+        bottom: 190px;
+        z-index: 3;
+      }
+
+      .middle-car img {
+        width: 150px;
+        height: 150px;
+        animation: cssAnimation 3s 2s forwards;
+        opacity: 0;
+        -webkit-animation-iteration-count: 1;
+      }
+
+      @keyframes cssAnimation {
+        to {
+          opacity: 1;
+        }
+
+        100% {
+          transform: translatey(100px) scale(1.5, 1.5);
+        }
+      }
+
+      .now-introducing-gloster {
+        left: 40%;
+        position: absolute;
+        top: 200px;
+        z-index: 3;
+      }
+
+      .now-introducing-gloster img {
+        width: 285px;
+        height: 65px;
+        animation: moveup 2s 2s linear normal;
+        opacity: 0;
+        -webkit-animation-iteration-count: 1;
+        animation-fill-mode: forwards;
+      }
+
+      @keyframes moveup {
+        100% {
+          transform: translatey(-120px) scale(2, 2);
+        }
+
+        to {
+          opacity: 1;
+        }
+      }
+    </style>
+  </head>
+  <body>
+    <div class="container">
```

Comment:
Improve your naming convetions check out BEM methodology.
https://getbem.com/naming/
https://css-tricks.com/bem-101/

ALso reduce the amount of nesting